Header img_header
LabeledBoundingBox[] bounding_boxes
